code-smells/indecent-exposure/after.js

```diff
@@ -0,0 +1,66 @@
+/* This smell indicates the lack of information hiding.
+The smell occurs when methods or classes that ought not be visible to clients are publicly visible to them.
+Exposing such code means that clients know about code that is unimportant or only indirectly important. 
+This contributes to the complexity of a design.
+This is a "Couplers" Code Smell type.
+*/
+
+const buildJail = () => {
+    // Understands prisoners treatment
+
+    const HOUR = 1000 * 60 * 60;
```

Comment:
I would like to have a quick cleancode conversation about what you said of `magic number`